src/main/java/com/comphenix/example/NbtFactory.java

```diff
@@ -52,7 +52,7 @@ private NbtFactory(){
 
                 // Prepare NBT
                 COMPOUND_CLASS = getMethod(0, Modifier.STATIC, offlinePlayer, "getData").getReturnType();
-                BASE_CLASS = Reflector.getClass(ClassType.NMS, "NBTBase");
+                BASE_CLASS = Reflector.getClass(ClassType.NMS, "nbt.NBTBase");
```

Comment:
does this work across versions?

Reply:
I am not sure what you mean by that? Nothing really changed, the `nbt.` will be stripped on earlier versions

Reply:
ah, my bad, i forgot that the `nbt.` gets stripped on earlier versions. that package prefix stripping is a bit hard to comprehend from the perspective of consumers... oh well

Reply:
The stripping code or the idea? The stripping code could probably handle some more love, but I couldn't think of an easier way to handle the different packages for now. Do you have any suggestions?